jdbc/src/java/org/apache/hive/jdbc/HiveDatabaseMetaData.java

```diff
@@ -855,19 +860,19 @@ public boolean nullPlusNonNullIsNull() throws SQLException {
   }
 
   public boolean nullsAreSortedAtEnd() throws SQLException {
-    throw new SQLFeatureNotSupportedException("Method not supported");
+    return false;
   }
 
   public boolean nullsAreSortedAtStart() throws SQLException {
-    throw new SQLFeatureNotSupportedException("Method not supported");
+    return false;
   }
 
   public boolean nullsAreSortedHigh() throws SQLException {
-    throw new SQLFeatureNotSupportedException("Method not supported");
+    return !getHiveDefaultNullsLast(connection.getConnParams().getHiveConfs());
```

Comment:
Shouldn't this be the opposite, i.e., isn't `hive.default.nulls.last` same as `nullsAreSortedHigh`?
```
Whether to set NULLS LAST as the default null ordering for ASC order.
```

Reply:
Right, fixed it.